.github/workflows/pr_review.yml

```diff
@@ -18,6 +18,7 @@ concurrency:
     'pull_request_review_comment' && 'pr_comment' || 'pr' }}
   cancel-in-progress: ${{ github.event_name != 'pull_request_review_comment' }}
```

Comment:
:robot: OpenAI

The `cancel-in-progress` key seems to be out of place and should be moved inside the `review` job definition.

```yaml
jobs:
  review:
    permissions: write-all
    cancel-in-progress: ${{ github.event_name != 'pull_request_review_comment' }}
```

23:
The `permissions` key with the value `write-all` seems overly permissive. It's recommended to only grant the necessary permissions for each job. Consider revising this to only grant the necessary permissions.

Overall, it would be helpful to have more context on the purpose of this workflow and the larger project in order to provide a more thorough review.

<!-- This is an auto-generated comment by OpenAI -->